sirepo/runner_daemon/docker_process.py

```diff
@@ -189,5 +189,9 @@ def __init__(self, container, backend_info):
 
 
 cfg = pkconfig.init(
-    dev_env_in_container=(False, bool, 'mount host dev environment into container'),
+    mount_dev_env_into_container=(
```

Comment:
Nit and don't fix now, just noting: when something returns a bool (like channel_in), you don't need to convert to a bool. If you are unsure, you can say `bool(expr)` if you care. If it is an inverse case, you can say `not bool(expr)`. Using an `if` introduces an unnecessary cognitive load.